README.md

```diff
@@ -40,6 +40,12 @@ the build generated will be placed in `dist` folder.
 
 `npm run test:integration`
 
+#### Configure the integration testing environment
+
+The integration testing environment is implemented with Test Containers and Docker Compose. The environment uses different environment variables, defined in a .env file, available in the _test/integration/environment_ folder.
+
+These environment variables can be updated as needed for integration testing. For example, we can specify the Dataverse image tag, corresponding to the Dataverse branch to test, through the DATAVERSE_BRANCH_NAME variable.
+
```

Comment:
I changed the DATAVERSE_BRANCH_NAME=develop and it raised this error after npm run test:integration

```
RUNNING INTEGRATION TESTS
Determining test suites to run...Running test containers...
Error: Jest: Got error running globalSetup - /Users/melina/GitHub/dataverse-client-javascript/test/integration/environment/setup.js, reason: test_dataverse Pulling 
test_dataverse Error 
Error response from daemon: manifest unknown

    at /Users/melina/GitHub/dataverse-client-javascript/node_modules/testcontainers/dist/src/docker-compose/functions/docker-compose-up.js:38:15
    at Generator.next (<anonymous>)
    at fulfilled (/Users/melina/GitHub/dataverse-client-javascript/node_modules/testcontainers/dist/src/docker-compose/functions/docker-compose-up.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

How can I test the module against develop?